library/build.gradle

```diff
@@ -22,6 +22,6 @@ android {
 }
 
 dependencies {
-    implementation 'com.android.support:appcompat-v7:27.0.2'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
```

Comment:
Are you updating this but not updating the support library in the main branch?

Reply:
yes. Trying not to open a can of worms.